packages/beacon-node/src/network/reqresp/ReqRespBeaconNode.ts

```diff
@@ -325,6 +360,8 @@ export class ReqRespBeaconNode extends ReqResp {
     this.onIncomingRequestBody({method: ReqRespMethod.Metadata, body: null}, peerId);
 
     const metadata = this.metadataController.json;
+
+    // TODO(fulu): is this still correct?  there are three versions of Metadata now...
```

Comment:
yes this is still correct, can remove TODO

Reply:
updated in #8128